tests/js/client/server_parameters/log-structured-params.js

```diff
@@ -0,0 +1,109 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
We must also add a test for the JSON-based log output, i.e. when using `--log.use-json-format=true` as a startup option.
This requires an extra test file.